include/aspect/plugins.h

```diff
@@ -354,7 +354,7 @@ namespace aspect
         (void)documentation;
         Assert (plugins != 0,
                 ExcMessage ("No postprocessors registered!?"));
-        Assert (name != "",
+        Assert (name != "unspecified",
```

Comment:
This assert shouldn't get hit, the individual models all throw their own exceptions if the user forgot to set a model name.